packages/lesswrong/server/notificationCallbacksHelpers.tsx

```diff
@@ -239,7 +239,7 @@ export const createNotification = async ({
     await notificationDebouncers[notificationType]!.recordEvent({
       key: {notificationType, userId},
       data: createdNotification.data._id,
-      timing: getNotificationTiming(notificationTypeSettings),
+      timing: getNotificationTiming(email),
```

Comment:
This is the most important file to review as this is what triggers notifications actually being send